### DISCUSSION TITLE
V2 fork and local development

### DISCUSSION THREAD

**Discussion by bartekpaczek** [medusajs/medusa#10772] [Q&A]
Hi,
I'm new to Medusa and thinking of building B2B platform with it. While going through documentation I have troubles understending on how to start separate project with it. I found documentation on forking for v1 here: https://docs.medusajs.com/v1/development/fundamentals/local-development, but documentation for v2 always referes to making changes in \src directory, but is not saying how to start a project, what to fork and how to build it. Is v1 documentation relevant?

**Comment by olivermrbl** [comment]
@bartekpaczek, I recommend reading through the [introduction and installation guide](https://docs.medusajs.com/learn). You create a new Medusa project by running `npx create-medusa-app@latest`.

**Reply by BloomingFlowers** [reply]
what if I try to research and develop medusajs locally, How could I run medusajs from source code?
I've tried the command "npx create-medusa-app@latest" ,after that I got a app with empty "src" folders and all the apis were requested from node_modules @medusa, that didn't work for me.
And I've tried to build from source code, which made some dependency errors like below:

@medusajs/admin-bundler:build:   The Yarn Plug'n'Play manifest forbids importing "@medusajs/ui-preset" here because it's not listed
@medusajs/admin-bundler:build:   as a dependency of this package:
@medusajs/admin-bundler:build:     ../../../../../../.pnp.cjs:121:31:
@medusajs/admin-bundler:build:       121 │         "packageDependencies": [\

Meanwhile we lack of dev or start script in package.json for run.

Appreciate to get docs about installation from source code
